code/modules/mob/living/carbon/human/human_species.dm

```diff
@@ -35,3 +35,7 @@
 
 /mob/living/carbon/human/synthetic_old/New(var/new_loc)
 	..(new_loc, "Early Synthetic")
+
+/mob/living/carbon/human/moth/New(var/new_loc)
+	moth_wings = pick(moth_wings_list - "Burnt Off")
+	..(new_loc, "Moth")
```

Comment:
You still have the same args as human, just do `/New(new_loc, new_species="Moth")`

Reply:
If I did that, wouldn't it set the wings after spawning their character, but then not update? Would you rather I add new_species="Moth" then call update_body, or keep it like this?